runtime/mangata-kusama/src/weights/extrinsic_weights.rs

```diff
@@ -0,0 +1,76 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-11-09 (Y/M/D)
+//! HOSTNAME: `ns5000592`, CPU: `Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz`
+//!
+//! SHORT-NAME: `extrinsic`, LONG-NAME: `ExtrinsicBase`, RUNTIME: `Mangata Development`
+//! WARMUPS: `10`, REPEAT: `100`
+//! WEIGHT-PATH: ``
+//! WEIGHT-METRIC: `Average`, WEIGHT-MUL: `1.0`, WEIGHT-ADD: `0`
+
+// Executed Command:
+//   ./target/release/mangata-node
+//   benchmark
+//   overhead
+//   --max-ext-per-block
+//   2000
+//   --execution
+//   native
+//   --chain
+//   dev
+
+use frame_support::{
+	parameter_types,
+	weights::{constants::WEIGHT_PER_NANOS, Weight},
+};
+
+parameter_types! {
+	/// Time to execute a NO-OP extrinsic, for example `System::remark`.
+	/// Calculated by multiplying the *Average* with `1.0` and adding `0`.
+	///
+	/// Stats nanoseconds:
+	///   Min, Max: 77_874, 78_235
+	///   Average:  77_996
+	///   Median:   77_994
+	///   Std-Dev:  62.2
+	///
+	/// Percentiles nanoseconds:
+	///   99th: 78_180
+	///   95th: 78_096
+	///   75th: 78_025
+	pub const ExtrinsicBaseWeight: Weight = Weight::from_ref_time(77_996).saturating_mul(WEIGHT_PER_NANOS.ref_time());
```

Comment:
did we just go from 186 to 77 ?
is this correct ? i think the default value from substrate is ~86

Reply:
it might, we have extra:
- inherent that persists data to runtime storage queue
- shuffling seed
- incoming txs validation
- transactions shuffling